love/src/components/Layout/Layout.jsx

```diff
@@ -131,8 +131,8 @@ class Layout extends Component {
   };
 
   componentDidUpdate = (prevProps, prevState) => {
-    if (this.props.config?.alarms && this.props.config.alarms !== prevProps.config?.alarms) {
-      const minSeverityNotification = this.props.config.alarms.minSeverityNotification?.trim().toLowerCase();
+    if (this.props.config?.content?.alarms !== prevProps.config?.content?.alarms) {
```

Comment:
I'm not familiar with this nomenclature of if, could you explain it to me please?